README.md

```diff
@@ -1,15 +1,106 @@
+   ```
+
+### Using helm
+
+```
+helm upgrade --install locust [FILEPATH-FROM-YOUR-LOCAL-MACHINE]ras-rm-performance-tests/_infra/helm/locust \
+    --namespace [TARGET-NAMESPACE] \
+    --set-string gcp.project=[TARGET-PROJECT] \
+    --set-string env=[TARGET-ENV] \
+    --set-string namespace=[TARGET-NAMESPACE] \
+    --set-string locust.loadtest.environment.test_respondents=8 \
+    --set-string locust.loadtest.environment.user_wait_time_min_seconds=2
+    --set-string locust.loadtest.environment.user_wait_time_max_seconds=3
+    --set-string locust.master.environment.LOCUST_USERS=8 \
+    --set-string locust.master.environment.LOCUST_SPAWN_RATE=1 \
+    --set-string locust.master.environment.LOCUST_RUN_TIME="60m" \
+    --set-string locust.loadtest.locust_locustfile=[LOCUST-CONFIG-FILE].py
+```
+
+These below commands are to be used when Performance is not the target project
+
+```
+    --set-string locust.loadtest.locust_host=http://frontstage.[TARGET-NAMESPACE].svc.cluster.local:9000 \
+    --set-string locust.loadtest.environment.case=http://case.[TARGET-NAMESPACE].svc.cluster.local:8080 \
+    --set-string locust.loadtest.environment.collection_exercise=http://collection-exercise.[TARGET-NAMESPACE].svc.cluster.local:8080 \
+    --set-string locust.loadtest.environment.collection_instrument=http://collection-instrument.[TARGET-NAMESPACE].svc.cluster.local:8080 \
+    --set-string locust.loadtest.environment.party=http://party.[TARGET-NAMESPACE].svc.cluster.local:8080 \
+    --set-string locust.loadtest.environment.sample=http://sample.[TARGET-NAMESPACE].svc.cluster.local:8080 \
+    --set-string locust.loadtest.environment.sample_file_uploader=http://sample-file-uploader.[TARGET-NAMESPACE].svc.cluster.local:8080 \
+    --set-string locust.loadtest.environment.survey=http://survey.[TARGET-NAMESPACE].svc.cluster.local:8080
+```
+
+Once completed, run:
+`helm uninstall locust --namespace [NAMESPACE] --ignore-not-found`
+
+### Database clear down between tests
+
+Firstly, whitelist you IP using the ras commands for the target environment. Then get the database credentials
+
+`export [TARGET-NAMESPACE]_PASSWORD_[TARGET-SERVICE]=$(kubectl get secret [DB-CREDENTIALS] -o json --namespace=[TARGET-NAMESPACE] | jq -r '.data."[PASSWORD-KEY]"' | base64 -d)`
+
+The above command will need to be replicated for all the schemas affected by the Locust tests. When using dev, this 
+will only be a single command.
+
+Once the passwords have been exported to your local machine, run the following:
+
+`psql postgresql://case:$[TARGET-NAMESPACE]_PASSWORD_[TARGET-SERVICE]@localhost:5432/[TARGET-SCHEMA-NAME] -U postgres -f [FILEPATH-FROM-YOUR-LOCAL-MACHINE]Services/ras-rm-cucumber/_infra/helm/acceptance-tests/files/[SCRIPT-NAME]`
+
+Again, the above command will need to be replicated for all the schemas affected by the Locust tests. When using dev, this 
+will only be a single command. The scripts that need running during multiple attempts are:
```

Comment:
Specify these can be found in the cucumber repo for reference

Reply:
Reference added.